src/status_im2/contexts/onboarding/identifiers/style.cljs

```diff
@@ -0,0 +1,15 @@
+(ns status-im2.contexts.onboarding.identifiers.style)
+
+(def page-container
+  {:flex 1})
+
+(def content-container
+  {:position :absolute
```

Comment:
Any particular reason for using `absolute` position & `top` instead of going with flexbox?

Reply:
here it's because it's just working with the view component for carousel. I can try refactor so it always sits below the carousel component. This was just a faster approach to get it set up